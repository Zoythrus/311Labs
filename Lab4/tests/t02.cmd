 text0.txt 0 6 
