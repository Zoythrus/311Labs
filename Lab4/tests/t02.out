0 3 5 1 4 6 2 7 8 9 10 
